src/Components/Card/Card.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import CardHeader from "./CardHeader";
+import CardBody from "./CardBody";
+import './CardStyles.css';
```

Comment:
please setup eslint with some settings and fix issues before the push
currently f.e. I see inconsistency in using single and double quotes